engine/app/components/citizens_advice_components/callout_component.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+module CitizensAdviceComponents
+  class CalloutComponent < ViewComponent::Base
+    attr_reader :type, :title
+
+    def initialize(type: nil, title: nil)
+      super
+      @type = type&.to_sym || :standard
```

Comment:
Why are we converting the type to a symbol here?

Reply:
Casting it, rather than converting it, right? Users are expected to pass a Symbol but if they pass a String it'll get cast back into a Symbol.

With this kind of thing, where the value gets rendered in the final HTML, you could have a holy war about whether String or Symbol is better practice because they both have merit but it really doesn't matter as long as you pick one and stick to it, since String is implemented the same ways as Symbol these days anyway (two identical strings will always occupy the same place in memory).

Reply:
I did have this a plain string, which was a little simpler. But convinced myself a symbol was more descriptive; good to know they are basically the same anyway.

Reply:
Gotcha - thanks!